src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,15 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+
+	public String getPrintableString(Printable... printables) {
+		StringBuilder s = new StringBuilder();
+		for(int i = 0; i < printables.length; i++){
+			s.append(printables[i].getPrintableString());
+			if(i != printables.length - 1){
+				s.append(", ");
```

Comment:
Doesn't this result in an extra `,` at the end? You could have used `StringJoiner` or `String.join` instead.